functions/AFQ_Create.m

```diff
@@ -268,11 +290,26 @@
         if ~exist(mrtrixdir,'dir'),mkdir(mrtrixdir);end
         % Get the lmax from the afq structure
         lmax = AFQ_get(afq,'lmax');
-        files = mrtrix_init(AFQ_get(afq, 'dt6 path',ii),lmax,mrtrixdir);
-        afq.files.mrtrix.csd{ii} = files.csd;
-        afq.files.mrtrix.wm{ii} = files.wm;
+        multishell = true;
+        files = AFQ_mrtrixInit(AFQ_get(afq, 'dt6path',ii), ...
+                               lmax,...
+                               mrtrixdir,...
+                               afq.software.mrtrixVersion, ...
+                               afq.params.track.multishell); % true/false
+        if ~multishell
+            afq.files.mrtrix.csd{ii} = files.csd;
+            afq.files.mrtrix.wm{ii} = files.wmMask;
+        else
+            afq.files.mrtrix.csd{ii} = files.wmCsd;
```

Comment:
Yes, let me finish with the revision and I will update. 
In not-multishell wm was the wm mask, but in multishell it is the csd of the wm. 
I was changing the names in order to make them as similiar as the previous version as possible, but it is going to be a mess. 
As I told this afternoon, please wait until I finish again, it is being impossible to make 5ttgen work in linux and I have to edit mrtrix python code now. 
Sorry about the delay.